File: AdSec/.NET/ApiVersion/ApiVersion.cs
using Oasys.AdSec;

namespace ApiVersion
{
    // This is the simplest possible example.
    //
    // If you can compile and run it then you've installed the API successfully.
    public static class ApiVersion
    {
        public static void Main()
        {
            System.Console.WriteLine("The AdSec API version is " + IVersion.Api());
        }
    }
}

File: AdSec/Python/ApiVersion.py
# Load the AdSec API

# Import modules
from Oasys.AdSec import IVersion

# This is the simplest possible example.
#
# If you can run this then you've installed the API successfully.
if __name__ == "__main__":
    print("This AdSec API version is " + IVersion.Api())
